src/types/block.go

```
@@ -13,6 +20,72 @@ type Block struct {
 	// TODO: Size ?
 }
 
+// https://bitcoin.org/en/developer-reference#coinbase
+func parseHeight(txin wire.TxIn) (int, error) {
```

Doesn't need to return an error

---

will fix